src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -3172,9 +3173,9 @@ void TLayout::layout(const Harmony* item, Harmony::LayoutData* ldata, LayoutCont
     ldata->setPos(positionPoint);
 }
 
-void TLayout::layout(Hook* item, LayoutContext&)
+void TLayout::layout(const Hook* item, Hook::LayoutData* ldata)
 {
-    Hook::LayoutData* ldata = item->mutLayoutData();
+    LD_INDEPENDENT;
     ldata->setBbox(item->symBbox(item->sym()));
```

Comment:
I'm wondering if we shouldn't consider `sym` as layout data, calculated based on the hook type (see Hook::setHookType).

Reply:
I think yes, `sym` is a layout data, but I think this change will be made in some future iteration..